js/listdates-min.js

```diff
@@ -1,53 +1,113 @@
 /* global moment, humanizeDuration */
+const datelistContent = document.getElementById('datelist-content');
+const datelistError = document.getElementById('datelist-error');
+const btnSortEvent = document.getElementById('btn-sort-event');
+const btnSortMonth = document.getElementById('btn-sort-month');
+const btnSortDuration = document.getElementById('btn-sort-duration');
```

Comment:
To connect to my comment below, since we always want to treat these buttons as a group, this is a good opportunity to package them up into a single array value rather than separate `btnSort*` values.

Reply:
so something like this? 
```
const buttons = _.map(["btn-sort-month", "btn-sort-event", ...], function (buttonName) {document.getElementById(buttonName)})
```

Reply:
This is fine, but I make the same arguments about 3 being too small. Yeah, it's not sexy to define 3 button constants, but fuck it's easy to understand what's going on.

Reply:
I was picturing more like:

```js
const buttons = ['btn-sort-event', 'btn-sort-month', 'btn-sort-duration'].map(document.getElementById);
```

Using a class instead is also an option, so we wouldn't have to maintain this when adding or removing buttons:

```js
const buttons = Array.from(document.querySelectorAll('.sort-button'));
```

Note that we're just using `Array.from()` here to be able to use array methods, since it's initially a `NodeList`.

Reply:
That reads okay to me! Also, glad to see map made it into ES6. I forget that most of the goodies from underscore/lodash are now native!